rally-domain/src/main/groovy/yakworks/rally/orgs/model/ContactSource.groovy

```diff
@@ -17,18 +17,19 @@ import yakworks.security.audit.AuditStamp
 @GrailsCompileStatic
 class ContactSource implements SourceTrait, RepoEntity<ContactSource>, Serializable {
 
-    static belongsTo = [contact: Contact]
+    //static belongsTo = [contact: Contact]
+    Long contactId
 
     //The id from the outside source or of the collection step, promise or some future workflow template record that generated this
```

Comment:
lets clean this up if we are refacoring. descrip is wrong here. verify this is needed.

Reply:
this comment does not apply, `//The id from the outside source or of the collection step, promise or some future workflow template record that generated this` has no relevance here.